cmd/libs/go2idl/make-client/output/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
@lavalamp Do we have that rule? It seems weird to have a doc.go file when there is only one or mainly one file for the package.